llama_stack/providers/utils/kvstore/config.py

```diff
@@ -106,7 +107,30 @@ def validate_table_name(cls, v: str) -> str:
         return v
 
 
+class MongoDBKVStoreConfig(CommonConfig):
+    type: Literal[KVStoreType.mongodb.value] = KVStoreType.mongodb.value
+    host: str = "localhost"
+    port: int = 27017
+    db: str = "llamastack"
+    user: str = None
+    password: Optional[str] = None
+    collection_name: str = "llamastack_kvstore"
+
+    @classmethod
+    def sample_run_config(cls, collection_name: str = "llamastack_kvstore"):
```

Comment:
you should also add a `**kwargs` parameter to sample run config (since the `distro_codegen.py` script invokes it with a "__distro__" parameter)